### DISCUSSION TITLE
TrueCharts

### DISCUSSION THREAD

**Discussion by silentnyte** [automatic-ripping-machine/automatic-ripping-machine#1126] [Q&A]
Howdy, when using the TrueCharts version of ARM, I have added a host mount and set the permissions to 777 586:568. I then changed the arm.yaml settings to point media, db, and logs to the new host mount location. When I start up ARM I can see that it creates the arm.db and when I insert a disc I can see it create a log file. But it is unable to find the dvd title and will not start ripping. As soon as I point the db and log back to /home/arm everything works.

Why can't I change the db and log location?

**Comment by microtechno9000** [comment]
Do you have a link to the version of ARM your using?

**Reply by silentnyte** [reply]
I think 2.6.70

**Comment by microtechno9000** [comment]
Sorry we don't support the implementation from TrueCharts, in that we are not the same developers.

Please raise this with the developer implementing this on TrueNAS. If there is an issue with ARM feel free to raise an issue and we can resolve.

https://truecharts.org/charts/stable/automatic-ripping-machine/
